python/run

```diff
@@ -0,0 +1,212 @@
+#!/usr/bin/env python3
```

Comment:
Should this file be called `run.py` instead?

Reply:
When it's an executable on linux, the file extension is optional. Did that to make the run task command cleaner.

Reply:
Fair enough! :-)